awscli/customizations/globalargs.py

```diff
@@ -66,6 +58,23 @@ def _resolve_endpoint_url(value):
     return value
 
 
+def resolve_verify_ssl(parsed_args, session, **kwargs):
+    SYSTEM_CA_BUNDLE = '/etc/pki/tls/certs/ca-bundle.crt'
```

Comment:
I like this idea of using the system ca cert, but I'd like to explore a more generic solution that's more cross platform (windows, Mac, etc.).

I think this should be possible.  Would you be ok with removing this for now, and then have a separate PR for supporting a "system" value?